crates/task-impls/src/vid.rs

```diff
@@ -355,6 +364,53 @@ where
                     ))
                     .await;
             }
+
+            HotShotEvent::TransactionsSequenced(payload, _metadata, view) => {
+                // encode transactions for disperse calculation
+                let encoded_txns = match payload.encode() {
+                    Ok(encoded) => encoded,
+                    Err(e) => {
+                        error!("Failed to encode the block payload: {:?}.", e);
+                        return None;
+                    }
+                };
```

Comment:
Is there no way to replace 6 lines of code with the `?` operator?

Reply:
Unfortunately, the `handle_event` function this is in returns an option. I could change the return type to a `Result<Option<>>` to pass the error and print it up the stack if wanted